community-operators/automation-broker/admin.clusterrolebinding.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Probably can't do anything with this until marketplace is using the `type: grpc` when creating CatalogSources in OLM. Let me know if it needs to be removed.

Reply:
We will not be supporting non CRDs / CVSs / Packages to be stored Quay app-registry for 4.0. The workaround is to add whatever resources your operator requires to the Marketplace manifest. Example: https://github.com/operator-framework/operator-marketplace/pull/101